examples/index.html

```diff
@@ -177,13 +177,13 @@
             <a-slide-camera position="0 3 0"></a-slide-camera>
         </a-slide>
         <a-slide>
-            <a-slide-camera position="2 3 0"></a-slide-camera>
+            <a-slide-camera position="2 3 0" easing="easeInElastic"></a-slide-camera>
```

Comment:
the easings added in this example are not meaningful (it was just random easing settings to test and I committed them without thinking to remove them). On the other hand it does show that easing works in the index demo, so it doesn't seem to hurt to keep it